packages/data/tests/unit/adapters/dimensions/elide-test.ts

```diff
@@ -0,0 +1,168 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import ElideDimensionAdapter from 'navi-data/adapters/dimensions/elide';
+import { DimensionColumn } from 'navi-data/adapters/dimensions/interface';
+import { RequestV2, AsyncQueryResponse, QueryStatus, RequestOptions } from 'navi-data/adapters/facts/interface';
+import DimensionMetadataModel from 'navi-data/models/metadata/dimension';
+
+const fakeResponse: AsyncQueryResponse = {
+  asyncQuery: { edges: [{ node: { id: '1', query: 'foo', status: QueryStatus.COMPLETE, result: null } }] }
+};
+
+module('Unit | Adapter | Dimensions | Elide', function(hooks) {
+  setupTest(hooks);
+
+  test('find', async function(assert) {
+    assert.expect(2);
+
+    const adapter: ElideDimensionAdapter = this.owner.lookup('adapter:dimensions/elide');
+    const originalFactAdapter = adapter['factAdapter'];
+    const TestDimensionColumn: DimensionColumn = {
+      columnMetadata: DimensionMetadataModel.create({
+        id: 'dimension1',
+        source: 'elideTwo',
+        tableId: 'table1'
+      }),
+      parameters: {
+        foo: 'bar'
+      }
+    };
+
+    const expectedRequest: RequestV2 = {
+      columns: [{ field: 'dimension1', parameters: { foo: 'bar' }, type: 'dimension' }],
+      filters: [
+        { field: 'dimension1', parameters: { foo: 'bar' }, type: 'dimension', operator: 'in', values: ['v1', 'v2'] }
+      ],
+      sorts: [],
+      table: 'table1',
+      limit: null,
+      dataSource: 'elideTwo',
+      requestVersion: '2.0'
+    };
+    const expectedOptions = {
+      timeout: 30000,
+      page: 6,
+      perPage: 24
+    };
+
+    //@ts-ignore
+    adapter['factAdapter'] = {
```

Comment:
Since we pull the fact adapter in via the container, can instead register a TestFactAdatper silimar to what was done here: https://github.com/yahoo/navi/blob/request-v2/packages/data/tests/unit/services/navi-dimension-test.ts#L53